BraveWallet/Blockies/Blockies.swift

```diff
@@ -99,6 +99,16 @@ class Blockies {
     }
     return image
   }
+  
+  func walletPanelBackground() -> LinearGradient {
```

Comment:
Doesn't necessarily need to be a function since it takes no args, and please rename this according to however you name the containing view. We already know its for wallet so that prefix is redundant, and it could technically be used in the future for any background, not specific to the panel so want to avoid naming it like that. 

Personally I kind of like:
- `BlockieMaterial`/`var material: LinearGradient { ... }`
- `BlockieBackground`/`var background: LinearGradient { ... }`

Reply:
addressed in [474d50a](https://github.com/brave/brave-ios/pull/5217/commits/474d50a054c56f6116c89a6a0c4013832280381d)